### PR TITLE
[POZNAN-2023] Initial event page

### PR DESCRIPTION
**Pull request by luprzybyl**
Initial page for DevOpsDays Poznań 2023 event. We're opening CFP and closing down organizational topics to provide dates and venue location.

### CODE REVIEW

File: data/events/2023-poznan.yml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+
+  - name: "Łukasz Przybył"
+    employer: "Sphera"
+    linkedin: "https://www.linkedin.com/in/lukasz-przybyl/"
+    twitter: "luprzybyl"
+    image: lukasz-przybyl.jpg
+  - name: "Jakub Jakubik"
+    employer: "Egnyte Poland"
```
Comment (jakubjakubik) [inline data/events/2023-poznan.yml:65]:
```suggestion
    employer: "Dealfront"
```

**Comment by jyee** [comment]
Hi @luprzybyl @jakubjakubik I sent an email to you via the poznan@devopsdays.org group. Core team had some questions.

**Comment by luprzybyl** [comment]
hi @jyee ,

I have responded to your e-mail some time ago.

In the meantime we have managed to finalize preparations and we have a fixed date and agenda.
I will close this pull request and open new one soon for better clarity.

**Comment by luprzybyl** [comment]
Linking new PR here: https://github.com/devopsdays/devopsdays-web/pull/13453

**Pull request closed without merging**